http/server_test.ts

```diff
@@ -1081,8 +1081,8 @@ Deno.test(
     };
     const listener = Deno.listen(listenOptions);
 
-    const onRequest = deferred();
-    const postRespondWith = deferred();
+    const onRequest = Promise.withResolvers<void>();
```

Comment:
Is having to define a `void` generic argument a bug?

Reply:
It might be - maybe it should be a default? Can you open an issue in `deno` repo?